jsx/Form.js

```diff
@@ -1736,6 +1736,165 @@ class LorisElement extends Component {
+      width: '100%',
+    };
+    const styleColumn = {
+      display: 'flex',
+      flexDirection: 'column',
+      alignSelf: 'flex-start',
+      marginRight: '10px',
+    };
+    const styleContainer = {
+      paddingTop: '7px',
+      cursor: 'pointer',
+    };
+    const styleLabel = {
+      margin: 0,
+      color: '#064785',
+      cursor: 'pointer',
+    };
+    const styleInput = {
+      display: 'inline-block',
+      margin: '0 5px 0 5px',
+      cursor: 'pointer',
+    };
+
+    let content = [];
+    for (let key in this.props.items) {
+      if (this.props.items.hasOwnProperty(key)) {
+        const item = this.props.items[key];
+        if (item.hasOwnProperty('value')) {
+          content.push(
+            <div key={item.id}
+                 style={styleColumn}>
+              <div style={styleContainer}>
+                <label htmlFor={item.id}
+                       style={styleLabel}
+                >
+                  {item.label}
+                </label>
+                <input
+                  type='radio'
+                  name={this.props.name}
+                  value={item.value}
+                  id={item.id}
+                  required={required}
+                  disabled={disabled}
+                  onChange={this.handleChange}
+                  style={styleInput}
+                />
+              </div>
+            </div>
+          );
+        }
+      }
+    }
+
+    layout.push(
+      <div key={this.props.name + '_key'}
+           style={styleRow}>
+        {content}
+      </div>
+    );
+
+    return layout;
+  }
+
+  render() {
+    let errorMessage = null;
+    let requiredHTML = null;
+    let elementClass = this.props.elementClass;
+    let required = this.props.required ? 'required' : null;
+
+    // Add required asterix
+    if (required) {
+      requiredHTML = <span className='text-danger'>*</span>;
+    }
+    // Add error message
+    if (this.props.errorMessage) {
+      errorMessage = <span>{this.props.errorMessage}</span>;
+      elementClass = this.props.elementClass + ' has-error';
+    }
+    // Generate layout
+    const layout = this.generateLayout();
+
+    return (
+      <div className={elementClass}>
+        <label className={'col-sm-3 control-label'}
+               htmlFor={this.props.id}>
+          {this.props.label}
+          {errorMessage}
+          {requiredHTML}
+        </label>
+        <div className={'col-sm-9'}>
+          {layout}
+        </div>
+      </div>
+    );
+  }
+}
+RadioElement.propTypes = {
+  id: PropTypes.string,
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  value: PropTypes.number.isRequired,
+  items: PropTypes.array.isRequired,
+  disabled: PropTypes.bool,
+  required: PropTypes.bool,
+  errorMessage: PropTypes.string,
+  elementClass: PropTypes.string,
+  onUserInput: PropTypes.func,
+};
+RadioElement.defaultProps = {
+  id: null,
+  name: null,
+  label: null,
+  value: 0,
+  items: [
```

Comment:
could we change "items" to "options" to be more in sync with the SelectElement prop?

Reply:
it can also be an object with key-value pairs. might make it more simple to access the "id", which could just be the key itself